Src/App/Router.php

```diff
@@ -1,31 +1,19 @@
 <?php
 namespace Aot\App;
 
-use Aot\Controller\PostController;
+use Aot\Controller\ApiController;
 
 class Router
 {
-    private $pc_instance;
-
-
-    public function __construct()
-    {
-        $this->pc_instance = new PostController();
-    }
 
     /**
-     * select the method
-     * @param  array $datas
-     * @return array
+     * lead to the controller
+     * @param $parameters
      */
-    public function method($datas = [])
+    public function route($parameters)
     {
-        if ($datas['method'] == 'show' && !array_key_exists('id', $datas)){
-            return $this->pc_instance->index($datas);
-        } elseif ($datas['method'] == 'show') {
-            return $this->pc_instance->show($datas);
-        } else {
-            throw new \Exception("not found");
-        }
+        $controller = new ApiController();
```

Comment:
This controller is hardcoded.
By doing that, you cannot add a new controller.
1°) You can take the decision to scope all http calls to `ApiController`.
2°) Or you can rely on an http parameters to select the controller to call